Homework_7/functional.py

```diff
@@ -0,0 +1,133 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# In[27]:
+
+
+import sys
+
+
+def sequential_map(*args):
+    '''Function sequential_map() applies number of functions to every item of iterable object
+    
+    Parameters: names of functions for sequential application and iterable object with values
+    *args:
+    function1, 
+    function2, 
+    ... , 
+    iterable
+    
+    Returns:
+    conteiner (list): the resulted iterable
+    '''
+    
+    *functions, conteiner = args
+    combined_function = func_chain(*functions)
+    conteiner = list(map(combined_function, conteiner))
+    return conteiner
+
+
+def consensus_filter(*args):
+    '''Function consensus_filter() applies number of filtering functions to each value in iterable. 
+    All functions shoud return bool
+    
+    Parameters:
+    *args:
+    function1,
+    function2,
+    ...,
+    iterable
+    
+    Returns:
+    conteiner (list): the resulted iterable
+    '''
+    
+    *functions, conteiner = args
+    for func in functions:
+        conteiner = list(filter(func, conteiner))
+    return conteiner
```

Comment:
Превратить элемент в список можно только в самом конце, результат filter можно передавать в другие filter'ы
```suggestion
        conteiner = filter(func, conteiner)
    return list(conteiner)
```